pdp/static/js/__test__/canada_ex_app.test.js

```diff
@@ -26,7 +26,6 @@ require('./globals-helpers').importGlobals([
     { module: 'js/pdp_filters.js', spread: true },
     { module: 'js/pdp_map.js', spread: true },
     { module: 'js/pdp_raster_map.js', spread: true },
-    { module: 'js/pdp_vector_map.js', spread: true },
```

Comment:
I see there are a bunch of test files where deleting this line is the only change, was it for something not actually used in the tests, and just present due to copy and paste test coding?

Reply:
My post-hoc diagnosis is copy-paste, or otherwise mistaken inclusion in the first place, as you say.